pkg/api/store/ignoretransitioningerror/store.go

```diff
@@ -0,0 +1,49 @@
+package ignoretransitioningerror
```

Comment:
Do you think it's better to name the package as `hpa`, and specifically handle HPA conditions? It looks like a rare case to be used by other resources.

Reply:
Just want to specify the store usage with the package name.